dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceOutputStream.java

```diff
@@ -29,6 +29,9 @@
 import org.hpccsystems.commons.errors.HpccFileException;
 import org.hpccsystems.commons.ecl.FieldDef;
 
+/**
+ * An output stream that uses the row service and ESP file creation APIs in HPCC to write to a specific file part.
```

Comment:
Minor, we should assume potential users of these APIs don't necessarily know the ESP acronym. Perhaps worthwhile to reference ESP as HPCC Systems web services?